test/local/routes/account.js

```diff
@@ -411,11 +411,11 @@ describe('/account/create', () => {
       args = mockMailer.sendVerifyCode.args[0]
       assert.equal(args[2].location.city, 'Mountain View')
       assert.equal(args[2].location.country, 'United States')
-      assert.equal(args[2].uaBrowser, 'Firefox')
-      assert.equal(args[2].uaBrowserVersion, '52')
-      assert.equal(args[2].uaOS, 'Mac OS X')
-      assert.equal(args[2].uaOSVersion, '10.10')
-      assert.strictEqual(args[2].uaDeviceType, undefined)
+      assert.equal(args[2].uaBrowser, 'Firefox Mobile')
+      assert.equal(args[2].uaBrowserVersion, '9')
+      assert.equal(args[2].uaOS, 'iOS')
+      assert.equal(args[2].uaOSVersion, '11')
+      assert.strictEqual(args[2].uaDeviceType, 'tablet')
```

Comment:
This unrelated test needed to change due to my changes to the mockDB `createSessionToken` method.